### PR TITLE
test(functional): update the ESR to 102.4.0

### PR DESCRIPTION
**Pull request by ashrivastava-qa**
## Because

`Updating functional tests to FF 102.4.0esr as ESR 91 is not supported anymore`

## This pull request

`Will update the FF to ESR 102.4.0`

## Issue that this pull request solves

Closes: #[FXA-4579](https://mozilla-hub.atlassian.net/browse/FXA-4579)

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ commands:
         default: 6
     steps:
       - browser-tools/install-firefox:
-          version: 78.9.0esr
+          version: 91.5.0esr
       - browser-tools/install-geckodriver:
           version: v0.29.1
```
Comment (pdehaan) [inline .circleci/config.yml:58]: Unrelated (and possibly worth tracking separately in Jira), but do we want to bump Geckodriver as well (here and L80 below, possibly others).
Looks like v0.29.1 was circa Apr 9, 2021 and v0.32.0 was released 15 days ago; per https://github.com/mozilla/geckodriver/releases

File: .circleci/config.yml [hunk 2]
```diff
@@ -53,7 +53,7 @@ commands:
         default: 6
     steps:
       - browser-tools/install-firefox:
-          version: 78.9.0esr
+          version: 91.5.0esr
```
Comment (pdehaan) [inline .circleci/config.yml:56]: Interesting! According to https://whattrainisitnow.com/, the current ESR is 102.4.0.
I can't find a decent ESR calendar/roadmap, apart from trolling the https://ftp.mozilla.org/pub/firefox/releases/ directory here (which says [91.9.1esr/](https://ftp.mozilla.org/pub/firefox/releases/91.9.1esr/) might be the latest 91.x ESR build).

File: .circleci/config.yml (hunk 1 above)
Comment (ashrivastava-qa) [reply .circleci/config.yml]: Upgraded to ESR 102.4.0 and Geckodriver 0.32.0

File: .circleci/config.yml (hunk 2 above)
Comment (ashrivastava-qa) [reply .circleci/config.yml]: Upgraded to ESR 102.4.0 and Geckodriver 0.32.0

**Review by pdehaan — APPROVED** [review]
R+! 💯

**Pull request closed without merging**